lstchain/tools/tests/test_tools.py

```diff
@@ -415,3 +414,101 @@ def test_index_srcdep_dl3_files(temp_dir_observed_srcdep_files):
             'EFFECTIVE AREA', 'ENERGY DISPERSION'
     ]:
         assert hdu_name in data.hdu_table['HDU_NAME']
+
+def test_add_scale_true_energy_in_irfs(temp_dir_observed_files, simulated_dl2_file):
+    """
+    Checking the validy of modified IRFs after scaling the True Energy by a factor.
+    """
+
+    from lstchain.tools.lstchain_create_irf_files import IRFFITSWriter
+    from gammapy.irf import EffectiveAreaTable2D, EnergyDispersion2D
+    import astropy.units as u
+
+    irf_file=temp_dir_observed_files / "fe_irf.fits.gz"
+    irf_file_mod = temp_dir_observed_files / "mod_irf.fits.gz"
+    config_file = os.path.join(os.getcwd(), "docs/examples/irf_dl3_tool_config.json")
+    config_file_mod = os.path.join(os.getcwd(), "docs/examples/irf_dl3_tool_config_mod.json")
```

Comment:
if the e_scale factor can be parsed through cmd line, I'd do that instead of using the file